### ISSUE TITLE
[Ripple] this refs.node.measure is not a function

### ISSUE THREAD

**Issue by Crash--** [xinthink/react-native-material-kit#24]
I'm currently implementing some inputs like Radio or Checkbox and I'm trying to use your Ripple component but I got an error... 

![capture d ecran 2015-10-01 a 12 03 53](https://cloud.githubusercontent.com/assets/1107936/10218079/7fac5dd4-6834-11e5-87c5-ea73a32069bb.png)

Here is my render function : 
```jsx
return  (
      <View style={style.input}>
        <Ripple ref="container"
          >
            <View style={style.rounded}></View>
          </Ripple>
        <Text style={style.label}>FOO</Text>
      </View>
    )
```

And here is the result of a console.log(this.refs.node) in MKTouchable file.

![capture d ecran 2015-10-01 a 12 07 32](https://cloud.githubusercontent.com/assets/1107936/10218177/1927e082-6835-11e5-9c84-54437c312c4f.png)

I missed something ?

**Comment by xinthink** [comment]
Would you please try to add a null-check like what it does in #10, I don't know if it's the same problem.
`Ripple` also used to built `Button`, should work fine in most case.

**Comment by Crash--** [comment]
I already tried that, but I did it again. 

I put this into MKTouchable.js 

```jsx 
componentDidMount(){
    requestAnimationFrame(() => {
          if (this.refs.node== null) {
           return;
         }
      const container = this.refs.node;  // un-box animated view
      container.measure((left, top, width, height) => {
        this.setState({dimen: {width, height}});
      });
    });
  }
```

But I have container.measure is not a function. 

So the container is not null but it has not any measure function. Weird :/

**Comment by xinthink** [comment]
`MKTouchable` is calling `measure` on a native component, what version of RN are you using?
`Ripple` is used in `Button` as well, does it work?

**Comment by Crash--** [comment]
I'm using RN 0.11.4 and... Button as not working neither! I got the same error. 
Just upgrading to 0.12.0-rc and... same result. 

I'll clean all the mess I've done and restart a new project to see if the error is still there.

**Comment by xinthink** [comment]
I'm thinking about replacing `measure` with `onLayout`, not just because this issue, but also the strange behavior of RN Android processing `borderRadius`

**Comment by xinthink** [comment]
should be fixed in v0.2.0 just released, changed to `onLayout` callbacks
